doc/api/assert.md

```diff
@@ -814,6 +814,72 @@ undefined, a default error message is assigned. If the `message` parameter is an
 instance of an [`Error`][] then it will be thrown instead of the
 [`AssertionError`][].
 
+## `assert.mustCall(fn[, exact])`
+<!-- YAML
+added: REPLACEME
+-->
+
+* `fn` {Function}
+* `exact` {number} **Default** `1`
+
+Returns a function that wraps `fn`. The wrapper function is expected to be
+called exactly `exact` times before the process exits. If the function is
+not called exactly `exact` times before the process exits, an error is thrown.
```

Comment:
Saying `thrown` here is a bit misleading. The `process.on('exit')` handler does not actually throw, it prints a summary of the functions that were not invoked. It's a subtle distinction but important.